frontend/src/BlogDetail/BlogDetail.js

```diff
@@ -1,80 +1,72 @@
-import React from 'react';
-import NavBar from '../Navbar/NavBar';
+import React, { useState, useEffect } from "react";
+import { useParams } from "react-router-dom";
+import NavBar from "../Navbar/NavBar";
+import API from "../API";
+import Comments from "../Comments/Comments";
+import Footer from "../Footer/Footer";
 
-function BlogDetail() {
-	return (
-		<div>
-			<div>
-				<NavBar />
-			</div>
-			<div className='ui raised very padded text container'>
-				<div>
-					<h4 className='ui left aligned sub header'>Date Created</h4>
-					<h1 className='ui left aligned header'>Article Title</h1>
-				</div>
-				<div className='ui hidden divider'></div>
-				<div className='ui left aligned text container segment '>
-					<div className='ui placeholder'>
-						<div className='image header'>
-							<div className='line'></div>
-							<div className='line'></div>
-						</div>
-						<div className='paragraph'>
-							<div className='line'></div>
-							<div className='line'></div>
-							<div className='line'></div>
-							<div className='line'></div>
-							<div className='line'></div>
-						</div>
-					</div>
-				</div>
+const getDate = (post) => {
+  const date = new Date(post.published_at);
```

Comment:
```suggestion
const getDate = (published_at) => {
  const date = new Date(published_at);
```
Just only pass what is needed. Dont pass everything, it will take up a memory.

Reply:
Fixed this sir. Thank you